src/webgpu/shader/validation/types/textures.spec.ts

```diff
@@ -109,10 +108,7 @@ Besides, the shader compilation should always pass regardless of whether the for
   )
   .fn(t => {
     const { format, access, comma } = t.params;
-    const isFormatValid = isTextureFormatUsableAsStorageFormatInCreateShaderModule(
```

Comment:
Note this is the last use of isTextureFormatUsableAsStorageFormatInCreateShaderModule

I would remove it, but a bunch of methods in format_info.ts refer to it in comments.